ipaserver/install/installutils.py

```diff
@@ -447,10 +447,11 @@ class DirectiveSetter(object):
     with DirectiveSetter('/path/to/conf') as ds:
         ds.set(key, value)
     """
-    def __init__(self, filename, quotes=True, separator=' '):
+    def __init__(self, filename, quotes=True, separator=' ', comment='#'):
```

Comment:
I like it, but where is ``comment`` argument used? I don't see it.

Reply:
So this requires either further docs or a switch to Augeas.

A lot of the values in the default ssl.conf are commented out which means that new values are appended to the end of the file, outside of the VirtualHost. This can still work but the context is wrong and could really go badly if someone defined other VirtualHost with different trust chains.

I added the comment bit so that a new value that was otherwise commented out could be added near where it should have been. This code is a bit fragile and doesn't account for extra spaces, etc. but it worked in this one small use-case.

Augeas would be more robust.